site/content/docs/developing/overrides/cdk.en.md

```diff
@@ -80,6 +80,7 @@ In this example, you can view how to:
 - Delete a resource property.
 - Create new resources.
 - Modify a property of an existing resource.
+- Delete a resource.
```

Comment:
But this action isn't part of this first example, right? 🤔

Reply:
Maybe delete this bullet point here, then change the below to 
"The following example showcases how you can delete a resource-- here, the Copilot-created default log group that holds service logs." 
or something like that?